modules/tinymce/src/core/test/ts/browser/dom/SelectionQuirksTest.ts

```diff
@@ -68,4 +69,16 @@ describe('browser.tinymce.core.dom.SelectionQuirksTest', () => {
     assertNormalizeCounter(1);
     TinyAssertions.assertSelection(editor, [ 0, 0 ], 0, [ 0, 0 ], 0);
   });
+
+  it('TINY-4550: Normalization should not run after selecting all with keyboard shortcut when there is only an image in the content', () => {
+    const editor = hook.editor();
+    resetNormalizeCounter();
+    editor.setContent('<p><img src="https://www.tiny.cloud/images/glyph-tinymce@2x.png" alt="" width="354" height="116"></p>');
+    TinySelections.setCursor(editor, [ 0 ], 1);
+    editor.shortcuts.add('meta+a', null, 'SelectAll');
+    const isMac = Env.os.isMacOS() || Env.os.isiOS();
+    TinyContentActions.keydown(editor, 65, { metaKey: isMac, ctrlKey: !isMac });
+    TinyContentActions.keyup(editor, isMac ? 224 : 17, { });
+    Assertions.assertEq('Selection node should be the p node', '<p><img src="https://www.tiny.cloud/images/glyph-tinymce@2x.png" alt="" width="354" height="116"></p>', editor.selection.getContent());
```

Comment:
You could use the TinyAssertions.assertSelection(editor, [0], 0, [0], 1) here to verify that the image is selected. When checking the selection state we usually use that or assertCursor.

Reply:
Done!